java/jmh/README.md

```diff
@@ -0,0 +1,18 @@
+# JMH Benchmarks for RocksJava
+
+These are JMH micro-benchmarks for RocksJava functionality.
```

Comment:
I (and probably others) am not familiar with JMH. Suggest:

> `These are micro-benchmarks for RocksJava functionality, using JMH (Java Microbenchmark Harness - https://openjdk.java.net/projects/code-tools/jmh/).`

Reply:
Okay, all done :-)